__tests__/app.test.js

```diff
@@ -53,12 +53,41 @@ describe('app', () => {
         })
     });
 
-})
+        })
 
-    })
 
+        describe('GET/api/articles', () => {
+            it('status: 200 successful request should return an array of articles ', () => {
+                return request(app)
+                    .get('/api/articles')
+                    .expect(200)
+                    .then(({ body }) => {
+                        expect(body.articles).toBeInstanceOf(Array)
+                        expect(body.articles).toHaveLength(12);
```

Comment:
Great test, It's great to see that you're checking the length of the array we get back before looping over it to check the properties to prevent any false passing tests here.